### PR TITLE
owo

### PR DESCRIPTION
**Pull request by sbxte**

### CODE REVIEW

File: scripts/config.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ exports.MUTED_PREFIX = '[white](muted)';
 exports.bannedWords = (function (words) {
     return words.map(function (word) { return (typeof word == "string" || word instanceof RegExp) ? [word, []] : [word[0], word.slice(1)]; });
 })([
-    "uwu",
-    "nig" + "ger", "nig" + "ga", "niger", "ni8" + "8ger",
+    "uwu", "owo", //lol
+    "nig" + "ger", "nig" + "ga", "niger", "ni8" + "8ger", //our apologies to citizens of the Republic of Niger
```
Comment (BalaM314) [inline scripts/config.js:26]: `.js` files are compiled from `.ts` files, editing the comments here has no effect (they will be gone next time i change anything else)

File: scripts/config.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const bannedWords:[word:string | RegExp, whitelist:string[]][] = (
 	(words:(string | string[] | RegExp)[]) =>
 		words.map(word => (typeof word == "string" || word instanceof RegExp) ? [word, []] : [word[0], word.slice(1)])
 )([
-	"uwu", //lol
+	"uwu","owo", //lol
```
Comment (BalaM314) [inline scripts/config.ts:7]: hmm

Comment (BalaM314) [reply scripts/config.ts:7]: no

Comment (BalaM314) [reply scripts/config.ts:7]: maybe

**Review by BalaM314 — COMMENTED** [review]
lets put it to a poll
i am not in favor

**Comment by Jurorno9** [comment]
NOOOO

**Pull request closed without merging**